transcom/mymove
# added the visible field for SIT Departure Date on the SIT dashboard a…

## [Agility ticket](tbd)
B-18355

## Summary

Is there anything you would like reviewers to give additional scrutiny?
Make sure the SIT departure date populates when there is a SIT departure date on the current SIT and shows "-" whenever there is no SIT departure date on the current SIT.

## Verification Steps for the Author

These are to be checked by the author.


## Verification Steps for Reviewers

These are to be checked by a reviewer.


### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/getting-started/application-setup/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/getting-started/development/testing)

### How to test

Prerequisite: Make sure there has been a SIT Departure date added to the current SIT by Prime and TOO approved the added SIT Departure Date.
1. Access the office side
2. Login as a TOO
3. Navigate to the MTO tab of the move
4. As long as there is an approved SIT Departure Date on the current SIT, the SIT Departure Date section should be populated with the correct date.. otherwise, there should be a "-".

### Frontend


### Backend


### Database

#### Any new migrations/schema changes:


## Screenshots
![Screenshot 2023-12-22 at 11 39 29 AM](https://github.com/transcom/mymove/assets/147739091/1382325c-36ef-4a3a-9926-92dac0698ffa)
